toolchain/gcc-9.5.0-05-arm-pic_crtstuff.patch

```diff
@@ -1,11 +1,16 @@
 diff -Naur gcc-9.3.0-orig/libgcc/config/arm/t-phoenix gcc-9.3.0/libgcc/config/arm/t-phoenix
 --- gcc-9.3.0-orig/libgcc/config/arm/t-phoenix	1970-01-01 01:00:00.000000000 +0100
 +++ gcc-9.3.0/libgcc/config/arm/t-phoenix	2021-08-23 13:40:28.290478307 +0200
-@@ -0,0 +1,7 @@
+@@ -0,0 +1,12 @@
 +# PhoenixRTOS uses relocatable ELFs on NOMMU platforms
-+# if this is ARM Cortex-M subvariant -> build libgcc/crtstuff with PIC
-+# ALSO: ensure data is not intertwined with .text, otherwise running XIP programs would fail
-+ifneq ($(findstring -m,$(filter -march=arm%, $(CC)))),)
```

Comment:
the most important change is that we remove `-march=` from the result (surprisingly `-m` is always present within `-march`), rewritten it into separate variables to make more readable.

`PS_` prefix added as this file is sourced in many Makefiles.